app/grandchallenge/components/models.py

```diff
@@ -881,6 +881,13 @@ class ComponentInterface(OverlaySegmentsMixin):
         ),
         validators=[JSONSchemaValidator()],
     )
+    example_value = models.JSONField(
+        blank=True,
+        null=True,
+        default=None,
+        help_text="Example JSON that complies with the additional JSON schema",
+        validators=[JSONSchemaValidator()],
```

Comment:
Why are you validating that this is a schema? This is just a JSON value.

Reply:
Fixed, my mistake.